src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/AvenirLight.woff") format("woff");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url("./fonts/AvenirHeavy.woff") format("woff");
+  font-weight: 700;
+  font-style: normal;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  font-family: AvenirHeavy, Arial, Helvetica, sans-serif;
+  font-weight: 900;
+  color: #3d4e61;
+  background-color: #fff;
+  border-radius: 4px;
+  height: 42px;
+  padding-left: 32px;
+}
+
+.input::-webkit-search-cancel-button {
+  display: none;
+}
+
+.input::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19.12px;
+  color: #3d4e61;
+}
+
+.input:hover {
+  outline: none;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  outline: none;
+  box-shadow: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.2) ;
+}
+
+:focus::-webkit-input-placeholder {
+  color: transparent;
+  text-shadow: none;
+}
+
+:focus::-moz-placeholder {
+  color: transparent;
+  text-shadow: none;
+}
+
+:focus:-moz-placeholder {
+  color: transparent;
+  text-shadow: none;
+}
+
+:focus:-ms-input-placeholder {
+  color: transparent;
+  text-shadow: none;
+}
+
+input[data-qa="keypress"]::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  color: #3d4e61;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+input[data-qa="keypress"] {
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 63px;
+  height: 70px;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.img {
+  position: absolute;
+}
+
+.imgLookingBig {
```

Comment:
You can leave only one class(imgLookingBig or imgLookingSmall) and move the styles to a common class .img, so as not to repeat yourself